File: Android/app/src/main/java/com/jayfunc/carpecast/NameGenerator.kt
package com.jayfunc.carpecast

object NameGenerator {
    private val adjectives = listOf(
        "Clever", "Fast", "Brave", "Silent", "Mighty", "Quick", "Happy",
        "Bright", "Cool", "Calm", "Fierce", "Gentle", "Lucky", "Proud"
    )

    private val nouns = listOf(
        "Fox", "Tiger", "Bear", "Eagle", "Wolf", "Lion", "Hawk",
        "Owl", "Panda", "Shark", "Falcon", "Dolphin", "Panther", "Leopard"
    )

    fun generate(): String {
        val adj = adjectives.random()
        val noun = nouns.random()
        return "$adj-$noun"
    }
}

File: Windows/NameGenerator.cs
using System;

namespace CarpeCast
{
    public static class NameGenerator
    {
        private static readonly string[] Adjectives = {
            "Clever", "Fast", "Brave", "Silent", "Mighty", "Quick", "Happy",
            "Bright", "Cool", "Calm", "Fierce", "Gentle", "Lucky", "Proud"
        };

        private static readonly string[] Nouns = {
            "Fox", "Tiger", "Bear", "Eagle", "Wolf", "Lion", "Hawk",
            "Owl", "Panda", "Shark", "Falcon", "Dolphin", "Panther", "Leopard"
        };

        public static string Generate()
        {
            var random = new Random();
            string adj = Adjectives[random.Next(Adjectives.Length)];
            string noun = Nouns[random.Next(Nouns.Length)];
            return $"{adj}-{noun}";
        }
    }
}
